features/step_definitions/test_steps.rb

```diff
@@ -204,7 +204,29 @@
   pending
 end
 
-Then(/^I select "([^"]*)" from the dropdown list with classname "([^"]*)"$/) do |name,classname|
-  browser.div(:class,classname).click
-  browser.select_list(:class => classname).div(:text => name).select
+
+Then(/^I click on the drop down box with div class "([^"]*)"$/) do |dropdown|
+  browser.div(:class,dropdown).click
+end
+
+
+Then(/^I select "([^"]*)" with index "([^"]*)" with div class "([^"]*)"$/) do |name, index, classname|
+
+end
+
+
+Then(/^I print the contents of dropdown box with the class name "([^"]*)"$/) do |classname|
+  browser.divs(:class => classname).each do |div|
+    puts div.text
+  end
+end
+
```

Comment:
```
Then(/^I print the text with the class name "([^"]*)"$/) do |classname|
  text = browser.div(:class => classname)
  puts text.text
```

Used this to identify the class names of the induvidual options in the drop down box.

Looks like all the class names below, just returns the text `Economy class`

![image](https://cloud.githubusercontent.com/assets/20741047/23114458/3b89b4c8-f765-11e6-8ad8-60f00a01b18d.png)

![image](https://cloud.githubusercontent.com/assets/20741047/23114491/5c820df6-f765-11e6-9875-435da9afff79.png)

![image](https://cloud.githubusercontent.com/assets/20741047/23114505/6d2aafdc-f765-11e6-8620-abacee456403.png)